src/api.py

```diff
@@ -1475,18 +1475,18 @@ def HandleListConnections(self):
         Returns bitmessage connection information as dict with keys *inbound*,
         *outbound*.
         """
-        if BMConnectionPool is None:
+        if network.BMConnectionPool is None:
```

Comment:
If we don't conditionally import the connection pool then this check isn't necessary anymore.